library/std/src/sys/windows/path/absolute.rs

```diff
@@ -0,0 +1,76 @@
+// Original implementation taken from crate "normpath".
+// Copyright 2020 "dylni" (https://github.com/dylni)
```

Comment:
I based it off this comment:
https://github.com/rust-lang/rust/blob/285fa7ecd05dcbfdaf2faaf20400f5f92b39b3c6/library/std/src/sys/unix/memchr.rs#L1-L2

This mostly clarifies that code was copied from "normpath" instead of being copied from libstd to the crate. I usually document when code is copied [in the copying crate](https://github.com/dylni/normpath#third-party-software).

Since [copyrights are already retained](https://github.com/rust-lang/rust/blob/285fa7ecd05dcbfdaf2faaf20400f5f92b39b3c6/COPYRIGHT#L9-L10), this doesn't say anything that's not already implied. I added it based on the precedent and to clarify the copy direction.

Reply:
Fair enough. I don't particular understand copyright which is why they make me nervous. Feel free to mark this as resolved.

Reply:
It's a good question. I don't completely understand it either. I'll leave the conversation open, since it's useful.